docs/roadmap.md

```diff
@@ -48,3 +48,49 @@ The development roadmap of Apache EventMesh is an overview of the planned featur
+|        [Kafka](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-kafka)        |    ✅    |    ✅    |
+|       [Pulsar](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-pulsar)       |    ✅    |    ✅    |
+|     [RabbitMQ](./design-document/connect/rabbitmq-connector)     |    ✅    |    ✅    |
+|         [HTTP](./design-document/connect/http-connector)         |    ✅    |    ⬜    |
+|         [JDBC](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-jdbc)         |    ⬜    |    ✅    |
+|       [Spring](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-spring)       |    ✅    |    ✅    |
+| [OpenFunction](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-openfunction) |    ✅    |    ✅    |
+|         [File](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-file)         |    ✅    |    ✅    |
+|                      Email                  |    ⬜    |    ⬜    |
+|        [Redis](./design-document/connect/redis-connector)        |    ✅    |    ✅    |
+|        [S3 File](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-s3)        |    ⬜    |    ✅    |
+|                    ClickHouse                    |    ⬜    |    ⬜    |
+|      [MongoDB](./design-document/connect/mongodb-connector)      |    ✅    |    ✅    |
+|   [Prometheus](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-prometheus)   |    ✅    |    ⬜    |
+|      [Knative](./design-document/connect/knative-connector)      |    ✅    |    ✅    |
+|      [Pravega](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-pravega)      |    ✅    |    ✅    |
+|         More connectors will be added...         |   N/A   |   N/A   |
+
+|                  Platform / Product                  | Source |   Sink   |
+|:------------------------------------------:|:------:|:------:|
+| [Feishu/Lark](./design-document/connect/lark-connector) |    ⬜    |    ✅    |
+|     [DingTalk](./design-document/connect/dingtalk-connector)     |    ⬜    |    ✅    |
+|        [WeCom](./design-document/connect/wecom-connector)        |    ⬜    |    ✅    |
+|       [WeChat](https://github.com/apache/eventmesh/tree/master/eventmesh-connectors/eventmesh-connector-wechat)       |    ⬜    |    ✅    |
+|                      GitHub                      |    ⬜    |    ⬜    |
+|                     ChatGPT                      |    ⬜    |    ⬜    |
+|        [Slack](./design-document/connect/slack-connector)        |    ⬜    |    ✅    |
+|         More connectors will be added...         |   N/A   |   N/A   |
+
+## Event Store Implementation Status
+
+| Service / Middleware | Ingress | Topic Management |
+|:------------------------:|:------:|:------:|
+| RocketMQ | ✅ | ✅ |
+| Kafka | ✅ | ✅ |
```

Comment:
Does 'Topic Management' here refer to management in eventmesh-admin or eventmesh-dashboard (dev)? If so, in previous eventmesh-admin, topic management of RocketMQ seems to be an empty implementation and Kafka doesn't seem to have it implemented, and in current eventmesh-dashboard, they have been deleted.

Reply:
The "Topic Management" refers to the `admin` package of each storage-plugin, which is used in the "Topic" page of the original eventmesh-dashboard:main.